src/components/Input/Input.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './index.css';
```

Comment:
This path did not work and I had to change it to import '../Navbar/index.css';

Reply:
Implemented changes: 
- [x] Made form labels capitalized
- [x] Errors are shown on top of the screen
- [x] For the notice message, I think it's correct. When you don't put anything at the first, nothing will popup but once you type a text and you try to delete it, it's going to remind you that the name should contain atleast 3 characters which is true. But at the beginning, if you decide to skip the name and start with other fields for example, the message will not popup.

Reply:
[Screencast from 04-05-2023 10:58:12 AM.webm](https://user-images.githubusercontent.com/50754458/230061425-d3c6cabd-339f-4562-8a17-c4a35cd14d1a.webm)